### PR TITLE
doc(deps): indicate that new Maven Repo is necessary

### PR DESCRIPTION
**Pull request by educhastenier**
if pulling bonita-server(-sp).jar

Relates to [RUNTIME-991](https://bonitasoft.atlassian.net/browse/RUNTIME-991)

### CODE REVIEW

**Review by vbotnari12 — APPROVED** [review]
OK for me.

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -171,9 +171,43 @@ Bonita Runtime produces a warning log if the filter is set and ignores it.
 
 The tenant creation method has been deprecated following our 2021.1 decision to deprecate the Multi-Tenants architecture.
 
+=== Potential need of declaring external Maven Repository
+
+From version 2022.1-u2, if you embed *bonita-server(-sp).jar* in your application, you need to declare additional Maven Repository in your pom.xml / build.gradle. This is due to Security Fixes that required the use of a dependency NOT present in default Maven Repository (aka Maven Central). To do so, add the following code:
```
Comment (tbouffard) [inline modules/ROOT/pages/release-notes.adoc]: 🔥 Adding repositories in pom.xml has been described as a bad practise for years
- https://blog.sonatype.com/2009/02/why-putting-repositories-in-your-poms-is-a-bad-idea/ (2009)
- https://maven.apache.org/repository-management.html

The remark here should at least advise usage of repository manager. This is not the only place where this should advice!

**Review by tbouffard — CHANGES_REQUESTED** [review]

**Comment by tbouffard** [comment]
ℹ️ the build of the preview is failing due to the configuration of the GitHub workflow.
For xref validation between component, we add the 2022.1 branch of bonita because the cloud component references it directly.
We don't have logic in the workflow to not add the 2022.1 branch when the PR branch is based on this branch. So the content of the 2022.1 and PR branches are almost duplicated leading to an Antora error
> FATAL (antora): Duplicate nav in 2022.1@bonita: modules/ROOT/build-a-process-for-deployment-nav.adoc
  1: modules/ROOT/build-a-process-for-deployment-nav.adoc in https://github.com/bonitasoft/bonita-doc.git (ref: 2022.1)
  2: modules/ROOT/build-a-process-for-deployment-nav.adoc in https://github.com/bonitasoft/bonita-doc.git (ref: fix/indicate_ehcache_need_for_new_repo)

We are working on this topic as part of https://github.com/bonitasoft/bonita-documentation-site/issues/326

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -171,9 +171,43 @@ Bonita Runtime produces a warning log if the filter is set and ignores it.
...
+
+[source,groovy]
+----
+repositories {
+    mavenCentral()
+    maven {
+        url 'https://repo.terracotta.org/maven2/'
+    }
+}
+----
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc:205]:
```suggestion
----

Of course, you can abstract this additional complexity by putting this configuration in your https://maven.apache.org/repository-management.html[enterprise repository manager] which acts a proxy to any external repositories you may have.
```

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -171,9 +171,45 @@ Bonita Runtime produces a warning log if the filter is set and ignores it.
...
+
+Of course, you can abstract this additional complexity by putting this configuration in your https://maven.apache.org/repository-management.html[enterprise repository manager] which acts a proxy to any external repositories you may have.
```
Comment (tbouffard) [inline modules/ROOT/pages/release-notes.adoc]: This applies to both maven and gradle so putting it before the examples will make the message more visible. Currently, it seems to me that it only applies to gradle.

ℹ️ The repository manager usage is not only for simplification, but if the Runtime Team agrees with the current wording, it's fine to me.

**Review by tbouffard — APPROVED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 3 above)
Comment (educhastenier) [reply modules/ROOT/pages/release-notes.adoc]: Ok, I'll take into consideration this first remark (and let the second one like that 😉 )

Comment (educhastenier) [reply modules/ROOT/pages/release-notes.adoc]: done

Comment (tbouffard) [reply modules/ROOT/pages/release-notes.adoc]: thanks

**Pull request merged**